wp-content/themes/oceanwp-child/assets/css/template-styles.css

```diff
@@ -1,25 +1,41 @@
-p{
-    font-size:16px !important;
+body {
+    font-family: 'Lato';
+    font-style: normal;
+    font-weight: normal;
+    font-size: 16px;
+    line-height: 150%;
 }
 
 h1{
-    font-size: 80px
+    font-style: normal;
+    font-weight: bold;
+    font-size: 80px;
+    line-height: 100%;
 }
 
 h2{
-    font-size:35px
+    font-style: normal;
+    font-weight: bold;
+    font-size: 35px;
+    line-height: 40px;
 }
 
 h3{
-    font-size:22px
+    font-style: normal;
+    font-weight: bold;
+    font-size: 22px;
+    line-height: 32px;
 }
 
 h4{
-    font-size:20px;
     font-family: 'Lato' !important;
```

Comment:
Does it need to be `!important`? What happens if it is not? I see it is not `!important` in the other headers.

Reply:
Okay, when I had customized the theme from front-end, I had set all the headers to Arvo but h4 is one of the headers having Lato so had to put `!important` to take effect.